eo-maven-plugin/src/test/java/org/eolang/maven/DummyCentral.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2022 Objectionary.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang.maven;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.function.BiConsumer;
+import org.apache.maven.model.Dependency;
+
+/**
+ * The class for emulating of Maven Central repository.
+ * DummyCentral creates an empty dependency jar file under the path.
+ *
+ * @since 0.28.11
+ */
+final class DummyCentral implements BiConsumer<Dependency, Path> {
+
+    @Override
+    public void accept(
+        final Dependency dependency,
+        final Path path
+    ) {
+        try {
+            Files.createDirectories(path);
+            Files.createFile(
+                path.resolve(
+                    String.format("%s-%s.jar", dependency.getArtifactId(), dependency.getVersion())
+                )
+            );
+        } catch (final IOException ex) {
+            throw new IllegalStateException(
+                String.format("DummyCentral can't save dependency %s by path %s", dependency, path),
```

Comment:
@volodya-lombrozo the information about the originator of the exception is in stacktrace. No need to add this noise. Just say `"Can't save '%s' to '%s'"`. In general, the shorter your messages, the better. They must be short, but contain maximum information.

Reply:
@volodya-lombrozo moreover, if I remove this class tomorrow, I most probably will forget to rename it in the text

Reply:
@yegor256 fixed